benchmark-builds/README.md

```diff
@@ -0,0 +1,13 @@
+## What kind of benchmarks ? :
+
+The purpose of this, is to bench the current rust version vs. the [original version](https://github.com/google/zopfli/blob/master/README).
+
+The measurement is done using [hyperfine](https://github.com/sharkdp/hyperfine).
+
+## How to :
```

Comment:
Could you remove the spaces before the punctuation characters in the section titles, and the colon after a question mark? Like this:

```suggestion
## What kind of benchmarks?

The purpose of this, is to bench the current rust version vs. the [original version](https://github.com/google/zopfli/blob/master/README).

The measurement is done using [hyperfine](https://github.com/sharkdp/hyperfine).

## How to
```